Sources/PayPal/Models/PayPalResult.swift

```diff
@@ -0,0 +1,21 @@
+#if canImport(PayPalCheckout)
+import PayPalCheckout
+#endif
+
+import Foundation
+
+/// The result of a PayPal payment flow.
+public struct PayPalResult {
+
+    /// The order ID associated with the transaction.
+    public let orderID: String
+
+    /// Intent, this will be `CAPTURE` or `AUTHORIZE` if the order token was generated using the `v2/` API.
+    public let intent: String
```

Comment:
I don't think we need INTENT. We _could_ have a separate convo about it ... but we aren't including it on the `CardResult` (https://github.com/paypal/iOS-SDK/blob/main/Sources/Card/Models/CardResult.swift). A merchant will have access to the intent themselves since they create the order on their own.